servers/parser-graphql-wrapper/src/datasources/parserApiUtils.ts

```diff
@@ -30,9 +30,9 @@ export const getAuthors = (authors): Author[] => {
 export const getImages = (images): Image[] => {
   return Object.keys(images).map((index: string): Image => {
     return {
-      imageId: parseInt(images[index].image_id),
-      width: parseInt(images[index]?.width) ?? null,
-      height: parseInt(images[index]?.height) ?? null,
+      imageId: parseIntOrNull(images[index].image_id),
```

Comment:
just noting this cannot be null

Reply:
agreed will followup in the am to make this code better and a few tests.